### PR TITLE
Check tcp listener stats when given an address

### PR DESCRIPTION
**Pull request by mattdoller**
We currently only record metrics from the unix listener status, but
the checks script supports both the unix and tcp listeners.  This
updates the metrics check so that if an address is supplied, we'll
check the tcp listener stats.

## Pull Request Checklist

**Is this in reference to an existing issue?**
no

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [x] Existing tests pass 

#### Purpose

- allow unicorn stat gathering through web address, instead of solely through socket file

### CODE REVIEW

**Comment by mattdoller** [comment]
The CI build seems to have failed because github was unavailable for when the build ran, and I don't have the ability to retry it.

**Comment by mattdoller** [comment]
ignore my previous message- just force-pushed an empty commit, build is now green

File: CHANGELOG.md
Comment (majormoses) [inline CHANGELOG.md:8]: In general the versioning and dating are left to the maintainers for several reasons:
- we may disagree on the impact of the change and therefore how its versioned, in this case it is an enhancement which is backwards compatible so it would be a minor bump.
- we date the release rather then when the contribution was submitted, PR review and acceptance has no SLA and is not guaranteed to be merged within the same day
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often
- you only versioned the changelog entry and did not [version the actual plugin itself](https://github.com/sensu-plugins/sensu-plugins-unicorn/blob/1.0.1/lib/sensu-plugins-unicorn/version.rb)

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

File: bin/metrics-unicorn.rb
Comment (majormoses) [inline bin/metrics-unicorn.rb:37]: I think we should name this `network_socket` as that is more accurate term to represent a `(ip|host):port` address conveys just the address and not the port. I think that this will reduce future confusion.

**Review by majormoses — CHANGES_REQUESTED** [review]
LGTM after updating the option name to be more accurate and descriptive

**Pull request closed without merging**
